### PR TITLE
Adding SCCLI documentation

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@praveenkumar @bexelbie comments?

File: README.rst [hunk 1]
```diff
@@ -3,8 +3,33 @@ adb-utils
 
 Utility and Service scripts for the `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_.
 
+adb-utils provides following content to ADB Vagrant box
+
+* OpenShift as a service
+
+  adb-utils uses OpenShift docker containers and provides systemd service unit file which inludes some setup scripts for the purpose. With the systemd unit file, OpenShift can be configured and run in ADB with just "systemctl start openshift" command.
```
Comment (bexelbie) [inline README.rst]: adb-utils uses a systemd service unit file to launch and setup a containerized version of OpenShift. With the systemd unit file, OpenShift can be configured and run in the ADB by issuing the following command: `systemctl start openshift`

File: README.rst [hunk 2]
```diff
@@ -3,8 +3,33 @@ adb-utils
...
+
+* SCCLI
+
+The idea behind providing SCCLI is to provide a way for users who just want to do vagrant ssh in to the ADB vagrant box and do their work. With this CLI they can setup required providers e.g. single node Kubernetes , OpenShift , Mesos marathan etc with single CLI commands.
```
Comment (bexelbie) [inline README.rst]: SCCLI provides a single point of entry for managing services inside of the ADB.  This is primarily of interest to users who use the ADB as a VM they access via `vagrant ssh`.

This CLI can clean the environment and ensure that the requested service is set up properly.

File: README.rst [hunk 3]
```diff
@@ -3,8 +3,33 @@ adb-utils
...
+
+With Release 1.0 we are providing interface for setting up OpenShift or single node Kubernetes.
```
Comment (bexelbie) [inline README.rst]: Currently SCCLI only provides an interface for managing OpenShift or a single node Kubernetes instance.

File: README.rst [hunk 4]
```diff
@@ -3,8 +3,33 @@ adb-utils
...
+
+The usage of sccli is as below::
+
+       $ sccli --help
+
+       usage: sccli [service_name] || [clean]
+       List of possible service_name:
+                k8s punchiest
```
Comment (bexelbie) [inline README.rst]: k8s openshift

File: README.rst [hunk 5]
```diff
@@ -3,8 +3,33 @@ adb-utils
...
+
+
+This is packaged as a RPM and included in `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_ version 1.7.0 or later
```
Comment (bexelbie) [inline README.rst]: These utilities and unit files are packaged as an RPM and included in `Atomic-Developer-Bundle (ADB) <https://github.com/projectatomic/adb-atomic-developer-bundle>`_ version 1.7.0 or later.

**Comment by LalatenduMohanty** [comment]
@bexelbie thanks. I have updated the PR cc @praveenkumar

**Comment by praveenkumar** [comment]
LGTM

**Pull request merged**